assets/js/blocks/attribute-filter/edit.js

```diff
@@ -180,6 +181,35 @@ const Edit = ( { attributes, setAttributes, debouncedSpeak } ) => {
 							} )
 						}
 					/>
+					<ToggleControl
+						label={ __(
+							'Filter button',
+							'woo-gutenberg-products-block'
+						) }
+						help={ () => {
+							if ( showFilterButton ) {
+								return __(
+									'Results will only update when the button is pressed.',
```

Comment:
Maybe "products" would be clearer than "results" here.

Reply:
Sounds good, done in 31ed0281a89691e49dc00d014f9c6030e22c0a92.